data/rayeesrather99.json

```diff
@@ -2,7 +2,13 @@
   "name": "Rayees Ahmad Rather",
   "displayStatsPublic": true,
   "type": "personal",
-  "bio": "A 20 year old kid **Learning Programming**. Exploration stage and tech enthusiastic",
+  "bio": "A 20 year old kid **Learner | Programmer**. tech enthusiastic",
+  "tags": ["Open Source", "Learner","Programmer", "Web3"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/Rayees15" },
```

Comment:
Are both the Twitter accounts yours, because there are links to 2 different accounts in line no. `8` & `16` !

If you've mistakenly placed a wrong url, kindly update it. 🛠

![image](https://user-images.githubusercontent.com/88786642/213631582-d2eb5ef8-0f58-4b7f-876b-b9c995b0fa7a.png)

Reply:
done, now it looks good

Reply:
> Are both the Twitter accounts yours, because there are links to 2 different accounts in line no. `8` & `16` !
> 
> If you've mistakenly placed a wrong url, kindly update it. 🛠
> 
> ![image](https://user-images.githubusercontent.com/88786642/213631582-d2eb5ef8-0f58-4b7f-876b-b9c995b0fa7a.png)

Not related to the PR @kumarsonsoff3 , How did you take this screenshot? 👀

Reply:
> Not related to the PR @kumarsonsoff3 , How did you take this screenshot? 👀

Windows snipping tool + https://tweetlet.net/image  == above screenShot! 

Also, there are lots of other tools similar to this with more amazing features. 😄